core-web/libs/portlets/dot-ema/src/lib/edit-ema-editor/components/ema-page-dropzone/ema-page-dropzone.component.html

```diff
@@ -0,0 +1,23 @@
+<!-- {{rows | json}} -->
+<div *ngFor="let row of rows" [ngStyle]="getStyle(row, 'transparent')" data-type="row">
+    <div
+        *ngFor="let column of row.columns"
+        [ngStyle]="getStyle(column, 'transparent')"
```

Comment:
I don't like to call methods in the template... maybe we can turn this into a pipe? @dotCMS/frontend thoughts?

Reply:
This is too specific and component bounded to be a pipe in my opinion

Reply:
[ngStyle]="column | dynamicStyle:'transparent'" I like this better!